src/drivers/mpu6000/mpu6000.cpp

```diff
@@ -596,6 +596,9 @@ MPU6000::init()
 	_gyro_scale.z_offset = 0;
 	_gyro_scale.z_scale  = 1.0f;
 
+	/* Prime _gyro with parents devid. */
+	_gyro->_device_id.devid = _device_id.devid;
```

Comment:
Why not initialise this straight in the constructor?
https://github.com/PX4/Firmware/pull/1715/files#diff-afefce4f4a086c7a7d23e52028f9ae1eR1850